scanpy_scripts/lib/_filter.py

```diff
@@ -30,10 +30,12 @@ def filter_anndata(
     logging.debug('--category=%s', category)
     logging.debug('--subset=%s', subset)
 
-    if gene_name:
+    if 'mito' not in adata.var.keys() and gene_name:
         try:
             gene_names = getattr(adata.var, gene_name)
-            adata.var['mito'] = gene_names.str.startswith('MT-')
+            k_mito = gene_names.str.startswith('MT-')
+            if k_mito.sum() > 0:
```

Comment:
Should we issue a warning here to highlight if no MT-genes where found so that the user is aware?

Reply:
Makes sense, warning added.